kylin/src/main/java/org/apache/zeppelin/kylin/KylinInterpreter.java

```diff
@@ -148,6 +146,22 @@ public HttpResponse prepareRequest(String sql) throws IOException {
     return httpClient.execute(postRequest);
   }
 
+  public String getProject(String cmd) {
```

Comment:
Thanks Jongyoul for the review. You are right, I check the code from JDBCInterpreter and copy the similar function. Could you point me some note level properties examples?